package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stepfinance/solana-market-aggregator",
-  "version": "1.3.0",
+  "version": "1.3.1",
```

Comment:
Want to avoid bumping version numbers inside of PRs. My release process for aggregator has been the following:
1. Merge changes.
2. Run `yarn version` against `main`. This updates the `package.json` version, creates a release commit, and creates a git tag automatically.
3. Push changes and created tags (`git push && git push --tags`)
3. Run `npm publish` against `main`.

I'm going to remove the package increment from this PR and then merge